tests/broker-engine/opentelemetry.robot

```diff
@@ -2,6 +2,7 @@
 Documentation       Engine/Broker tests on opentelemetry engine server
 
 Resource            ../resources/import.resource
+Library             ../resources/Agent.py
```

Comment:
does this should be in import.ressource file to be available everywhere ?

Reply:
no it is used only in this file